recipes/ipytest/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "ipytest" %}
+{% set version = "0.5.0" %}
+{% set arch = "py2.py3" %}
+{% set fn = "{}-{}-{}-none-any.whl".format(name, version, arch) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/{{ arch }}/{{ name[0] }}/{{ name }}/{{ fn }}
+  sha256: c25e09654b2b431047ab7f381333188be2e8fc66146f1f2aed44dc105128b074
+  
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install {{ fn }} --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
```

Does this not depend on `ipython` and `pytest`?
```suggestion
 - ipython
 - pytest
 
```

---

Pip installing the package doesn't install ipython, but it does seem like the package is a bit useless without it.

---

pytest is also not entirely a requirement. This package can be used to run unit tests using the built-in python unit test runner, but that functionality is deprecated.

---

Given that usage without pytest is deprecated and the package fails if pytest is included but not ipython, it makes sense to add both dependencies